### PR TITLE
Updated and simplified some of the mobile iOS docs

### PR DESCRIPTION
**Pull request by mbruin-NR**
Updated the mobile iOS/tvOS docs to simplify the use instructions and fix an issue with dsym upload build step

Please follow [conventional commit standards](https://www.conventionalcommits.org/en/v1.0.0/)
in your commit messages and pull request title.

## Give us some context
https://issues.newrelic.com/browse/NR-67768
https://github.com/newrelic/newrelic-ios-agent-spm/issues/3#issuecomment-1413589865

Some of the docs for setting up the agent and automatically uploading the dsym as a build step where out of date and needed updating. In the i18n folder I did what I could but would need some help on bringing the same changes there. Thank you, let me know if there are any questions or concerns.

### CODE REVIEW

**Comment by a-sassman** [comment]
Hi @mbruin-NR , can you please sign the SLA agreement?

**Review by cdillard-NewRelic — APPROVED** [review]
👏

File: src/content/docs/mobile-monitoring/new-relic-mobile-ios/installation/ios-manual-installation.mdx [hunk 1]
```diff
@@ -170,7 +170,7 @@ If you have trouble getting your app to crash, the New Relic agent provides an A
 
 ## Optional: Change the logging level [#logging]
 
-By default the New Relic agent will log at the `info` level. Increasing the log level to `verbose` or higher is only encouraged for debugging and not for release builds. 
+By default the New Relic agent will log at the `info` level. Increasing the log level to `verbose` or higher priority is only encouraged for debugging and not for release builds. Verbose level or higher priority will log all API interactions with the New Relic server.
```
Comment (cdillard-NewRelic) [inline src/content/docs/mobile-monitoring/new-relic-mobile-ios/installation/ios-manual-installation.mdx:173]: Can you please add this change to the other places in the docs where we warn about verbose priority level or higher?

File: src/content/docs/mobile-monitoring/new-relic-mobile-ios/configuration/ios-tvos-crash-reporting.mdx [hunk 2]
```diff
@@ -31,6 +31,7 @@ For more information, see [Retrieve and download dSYMs](/docs/mobile-monitoring/
 
 Crash reporting is enabled by default, but there are some circumstances where it will be disabled:
 
+* **If running in simulator:** A build running through the simulator cannot be symbolicated.
```
Comment (a-sassman) [inline src/content/docs/mobile-monitoring/new-relic-mobile-ios/configuration/ios-tvos-crash-reporting.mdx:34]: @cdillard-NewRelic Can we make this into a full phrase? i.e., "If the app is running in a simulator:"

File: src/content/docs/mobile-monitoring/new-relic-mobile-ios/get-started/new-relic-ios-compatibility-requirements.mdx [hunk 3]
```diff
@@ -115,7 +115,7 @@ Make sure your iOS app meets these requirements:
       </td>
 
       <td>
-        To take advantage of New Relic's iOS features, make sure you have the latest version of [Xcode](/docs/mobile-monitoring/new-relic-mobile-ios/install-configure/upgrade-new-relic-mobiles-ios-sdk).
+        To take advantage of New Relic's iOS features, make sure you have the latest version of **[Xcode](https://developer.apple.com/documentation/xcode-release-notes)**.
```
Comment (a-sassman) [inline src/content/docs/mobile-monitoring/new-relic-mobile-ios/get-started/new-relic-ios-compatibility-requirements.mdx:118]:
```suggestion
        To take advantage of New Relic's iOS features, make sure you have the [latest version of Xcode](https://developer.apple.com/documentation/xcode-release-notes).
```

File: src/content/docs/mobile-monitoring/new-relic-mobile-ios/installation/cocoapods-installation.mdx [hunk 4]
```diff
@@ -62,11 +62,24 @@ These procedures to configure your iOS app with CocoaPods and Objective-C also a
...
+   ```   
+   SCRIPT=`/usr/bin/find "${SRCROOT}" -name run-symbol-tool | head -n 1`
+   /bin/sh "${SCRIPT}" "APP_TOKEN"
+   ```
+ - ### **iOS Agent 7.3.8 or before:**
    ```
    SCRIPT=`/usr/bin/find "${SRCROOT}" -name newrelic_postbuild.sh | head -n 1`
    /bin/sh "${SCRIPT}" "APP_TOKEN"
    ```
+ - **OPTIONAL:** Add the following lines to your build script above the existing lines to skip symbol upload during debugging.
```
Comment (a-sassman) [inline src/content/docs/mobile-monitoring/new-relic-mobile-ios/installation/cocoapods-installation.mdx:76]:
```suggestion
 - **OPTIONAL:** Add the following lines to your build script above the existing lines to skip symbol upload during debugging:
```

File: src/content/docs/mobile-monitoring/new-relic-mobile-ios/installation/cocoapods-installation.mdx [hunk 5]
```diff
@@ -62,11 +62,24 @@ These procedures to configure your iOS app with CocoaPods and Objective-C also a
      The agent must be on the first line of `didFinishLaunchingWithOptions` and run on the main thread to ensure proper instrumentation. Starting the call later, on a background thread, or asynchronously can cause unexpected or unstable behavior.
    </Callout>
 6. Add a build script to your target's **Build Phases**. Ensure the new build script is the very last build script. Then paste the following, replacing `APP_TOKEN` with your [application token](/docs/mobile-apps/viewing-your-application-token):
-
+     
+ - ###  **iOS Agent 7.4.0+:**
```
Comment (a-sassman) [inline src/content/docs/mobile-monitoring/new-relic-mobile-ios/installation/cocoapods-installation.mdx:66]:
```suggestion
 - ###  **iOS Agent 7.4.0 or higher:**
```

File: src/content/docs/mobile-monitoring/new-relic-mobile-ios/installation/cocoapods-installation.mdx [hunk 6]
```diff
@@ -62,11 +62,24 @@ These procedures to configure your iOS app with CocoaPods and Objective-C also a
...
+   ```   
+   SCRIPT=`/usr/bin/find "${SRCROOT}" -name run-symbol-tool | head -n 1`
+   /bin/sh "${SCRIPT}" "APP_TOKEN"
+   ```
+ - ### **iOS Agent 7.3.8 or before:**
```
Comment (a-sassman) [inline src/content/docs/mobile-monitoring/new-relic-mobile-ios/installation/cocoapods-installation.mdx:71]:
```suggestion
 - ### **iOS Agent 7.3.8 or lower:**
```

File: src/content/docs/mobile-monitoring/new-relic-mobile-ios/installation/cocoapods-installation.mdx [hunk 7]
```diff
@@ -88,17 +101,13 @@ These procedures to configure your iOS app with CocoaPods and Swift also appear
    ```
    open App.xcworkspace
    ```
-4. Create a bridging header:
-
-   1. Select **File > New > File > Objective-C File**.
-   2. Name the file `Placeholder.m`, then select **Next**.
-   3. Select **Create**, and confirm Xcode's prompt to generate the bridging header.
-5. In your `APP_NAME-Bridging-Header.h`, add the New Relic header:
+4. In your `AppDelegate.swift`, add the New Relic header:
```
Comment (a-sassman) [inline src/content/docs/mobile-monitoring/new-relic-mobile-ios/installation/cocoapods-installation.mdx:104]:
```suggestion
1. In your `AppDelegate.swift`, add the New Relic header:
```

File: src/content/docs/mobile-monitoring/new-relic-mobile-ios/installation/cocoapods-installation.mdx [hunk 8]
```diff
@@ -88,17 +101,13 @@ These procedures to configure your iOS app with CocoaPods and Swift also appear
...
+   (If your app is written in SwiftUI, follow these [instructions](/docs/mobile-monitoring/new-relic-mobile-ios/troubleshoot/swiftui-appdelegate) to add an AppDelegate to your project.)
```
Comment (a-sassman) [inline src/content/docs/mobile-monitoring/new-relic-mobile-ios/installation/cocoapods-installation.mdx:105]:
```suggestion
```

File: src/content/docs/mobile-monitoring/new-relic-mobile-ios/configuration/ios-tvos-crash-reporting.mdx (hunk 2 above)
Comment (cdillard-NewRelic) [reply src/content/docs/mobile-monitoring/new-relic-mobile-ios/configuration/ios-tvos-crash-reporting.mdx:34]: Yeah your edit plus one clarification in the last part looks good. 👍 
```suggestion
* **If the app is running in a simulator:** An app running through the simulator cannot be used with crash reporting. The simulator crashes cannot be symbolicated.
```

File: src/content/docs/mobile-monitoring/new-relic-mobile-ios/installation/ios-manual-installation.mdx (hunk 1 above)
Comment (a-sassman) [reply src/content/docs/mobile-monitoring/new-relic-mobile-ios/installation/ios-manual-installation.mdx:175]: I added it to the two tvos pages.

**Review by a-sassman — APPROVED** [review]

**Comment by a-sassman** [comment]
@cdillard-NewRelic @mbruin-NR Let me know when this is ready to be published - thanks!

**Comment by mbruin-NR** [comment]
This can be published asap. Thank you for your edits!

**Comment by a-sassman** [comment]
Just waiting on the build service check to complete and I'll merge. Thanks!

File: src/content/docs/mobile-monitoring/new-relic-mobile-ios/installation/ios-manual-installation.mdx [hunk 9]
```diff
@@ -37,71 +37,80 @@ Continue with the steps to configure New Relic.
...
+        </th>
+      </tr>
+    </thead>
+      <tbody>
+      <tr>
+        <td>
+          Objective-C
+        </td>
+        <td>
+          1. To start the agent, import the New Relic header by adding `#import <NewRelic/NewRelic.h> at the top of your `AppDelegate.m`.
```
Comment (bradleycamacho) [inline src/content/docs/mobile-monitoring/new-relic-mobile-ios/installation/ios-manual-installation.mdx:66]:
```suggestion
          1. To start the agent, import the New Relic header by adding `#import <NewRelic/NewRelic.h>` at the top of your `AppDelegate.m`.
```

**Pull request merged**